lib/gracenote/http.rb

```diff
@@ -26,10 +26,14 @@ def agent
       @agent ||= begin
         Faraday.new(url: host) do |conn|
           conn.response :xml,  :content_type => /\bxml$/
-          conn.use Faraday::Conductivity::ExtendedLogging, logger: ::Logger.new('log/faraday.log')
+          conn.use Faraday::Conductivity::ExtendedLogging, logger: ::Logger.new("log/faraday.log") if log_enabled?
```

Comment:
__[rubocop]__
* Line is too long. [114/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)